doc/abrt-action-install-debuginfo.txt

```diff
@@ -30,17 +33,25 @@ OPTIONS
    Path to tmp directory. Default: /tmp/abrt-tmp-debuginfo-RANDOM_SUFFIX
 
 --cache::
-   Path to cache directory. Default: /var/cache/abrt-di
+   Path to cache directory. Default: /var/cache/abrt-dir
```

Comment:
“di” stands for “debuginfo”, so the original was correct.

Reply:
But I see that the actual script has that mistake.

Reply:
@ernestask You are right. I was misled by the path in the help message. Fixed both.